builder/oracle/bmcs/artifact.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2017 Oracle America, Inc.
+// The contents of this file are subject to the Mozilla Public License Version
+// 2.0 (the "License"); you may not use this file except in compliance with the
+// License. If a copy of the MPL was not distributed with this file, You can
+// obtain one at http://mozilla.org/MPL/2.0/
+
+package bmcs
+
+import (
+	"fmt"
+	client "github.com/mitchellh/packer/builder/oracle/bmcs/client"
```

Comment:
@dave2 We don't currently have an official Go SDK so we've included a minimal implementation of the API operations required for Packer. It's likely that we'll factor it out, and follow the pattern you described, once an official SDK is available.